mvc/src/main/java/com/interface21/webmvc/servlet/mvc/tobe/AnnotationHandlerMapping.java

```diff
@@ -27,13 +24,20 @@ public AnnotationHandlerMapping(final Object... basePackage) {
         this.handlerExecutions = new HashMap<>();
     }
 
+    @Override
     public void initialize() {
         for (Object packageName : basePackage) {
-            scanControllers(packageName);
+            final Reflections reflections = new Reflections(packageName);
+            final ControllerScanner controllerScanner = new ControllerScanner(reflections);
+            final Map<Class<?>, Object> controllersClasses = controllerScanner.getControllers();
+            for (Object controllerInstance : controllersClasses.values()) {
+                registerControllerMethods(controllerInstance);
+            }
         }
         log.info("Initialized AnnotationHandlerMapping!");
     }
 
+    @Override
     public Object getHandler(final HttpServletRequest request) {
         final String url = request.getRequestURI();
```

Comment:
```suggestion
    public HandlerExecution getHandler(final HttpServletRequest request) {
        final String url = request.getRequestURI();
```

타입 안정성과 가독성을 위해 반환 타입을 더 구체적으로 명시하는거 어떨까요?

Reply:
그게 좋겠네요! 수정했습니다👍 

[refactor: AnnotationHandlerMapping getHandler 반환 타입 변경](https://github.com/dye0p/java-mvc/commit/76bc2c50c912ce74f7cf1bf34833bcce760cefc5)